### PR TITLE
Feedback please: Reorganise the top level navigation and merge some content

### PR DESCRIPTION
**Pull request by louzoid-gds**
Over Christmas, I attempted to alter the GDS Way to use the tech docs multipage option. Preview available here:
https://louises-gds-way.cloudapps.digital/

Although this was received well, I had to alter the structure of some pages to achieve a decent navigation experience. This was far from ideal.

This attempt retains the single page nav option but uses a custom built nav on the homepage to surface the extent of the content on the GDS way. Other minor amends have been made to merge content where replication exists and to improve navigation. These can be viewed in the commit history. 

Preview of the new structure (this PR) available here:
https://louises-gds-way-v2.cloudapps.digital/

Changes to be aware of:
- Custom nav created on home page and programming lang style guide page
- Publish open source code now included on how to store source code
- Git style guide now included on how to store source code
- How to review code' page not been included (issue #399)
- Check if a service is ready for production has been removed (needs content review)
- Cyber and RE pages now under Technical operations umbrella page
- RSpec page merged with Ruby style guide page

Feedback either here or in person very welcome!

### CODE REVIEW

File: source/standards/technical-operations.html.md.erb [hunk 1]
```diff
@@ -1,10 +1,33 @@
...
+# Technical operations team
+
+The Technical operations team focuses on improving how we run services and make decisions about technology while supporting the other service teams within GDS.
+
+We work with GDS technology teams to:
+
+* provide a common set of tools to support common problems
+* integrate and configure those tools
+* build and maintain a secure platform making it easy for teams to build services
+* provide Security Operations, Engineering and Architecture support to other teams in GDS
```
Comment (richardTowers) [inline source/standards/technical-operations.html.md.erb]: I think we should probably include user support in this, if we're going to expand the RE / Cyber section into the whole of techops.

```suggestion
* provide Security Operations, Engineering and Architecture support to other teams in GDS
* provide a first line support function for end users - engaging with them, understanding their problems, and helping resolve them
```

(copy stolen from the techops weeknote)

File: source/standards/technical-operations.html.md.erb [hunk 2]
```diff
@@ -1,10 +1,33 @@
...
+Reliability Engineering provide a shared platform to distribute tools and services. This includes support for:
+
+* Amazon Web Services (AWS) and the GOV.UK PaaS infrastructure
+* [Logit][], for storing and querying infrastructure and application logs
+* [Prometheus][], for running an operational metrics service
+
+You can read more about Reliability Engineering in [our documentation][].
+
+You also can contact Reliability Engineering by email using [reliability-engineering@digital.cabinet-office.gov.uk][] or using the [#reliability-eng Slack channel][].
+
```
Comment (richardTowers) [inline source/standards/technical-operations.html.md.erb]:
```suggestion

## User Support

User Support provide a first line responses to end user queries and feedback. They are responsible for engaging with users, understanding their problems and helping resolve them.

User Support also manage the configuration of GDS' central ticketing system, which uses Zendesk.
```

File: config/tech-docs.yml [hunk 3]
```diff
@@ -31,3 +31,10 @@ github_repo: alphagov/gds-way
 # Ownership for page reviews
 default_owner_slack: '#gds-way'
 owner_slack_workspace: gds
```
Comment (erino) [inline config/tech-docs.yml]: What about adding `collapsible_nav: true`. To create a top level collapsible nav as per https://tdt-documentation.london.cloudapps.digital

https://github.com/alphagov/tdt-documentation/blob/master/config/tech-docs.yml#L28

File: source/standards/technical-operations.html.md.erb (hunk 1 above)
Comment (louzoid-gds) [reply source/standards/technical-operations.html.md.erb]: Discussed offline and we'll save sorting this out for another later PR :)

File: config/tech-docs.yml (hunk 3 above)
Comment (louzoid-gds) [reply config/tech-docs.yml]: I tried this and it looks pretty good on the homepage but doesn't work on the lower level ages as all the content is by default hidden under the collapsed nav

Screenshot 2020-02-19 at 08 47 58

Comment (erino) [reply config/tech-docs.yml]: Good point, I didn't check lower level pages.

**Comment by stephenharker** [comment]
I think it looks much better. I think we should make this one live :)

**Pull request merged**